backend/src/v1/complaint/complaint.service.ts

```diff
@@ -651,6 +651,7 @@ export class ComplaintService {
     const officeQuery = await this._cosOrganizationUnitRepository
       .createQueryBuilder("cos_geo_org_unit")
       .where("cos_geo_org_unit.zone_code = :zone", { zone })
+      .andWhere("cos_geo_org_unit.administrative_office_ind = :value", { value: false })
```

Comment:
I'm not sure this is quite the right logic for this.   

For example, on the Admin/User page we want to see all users (this is a change that needs to be made)   in that case we'd want to include all values.   

I doubt there is a scenario where we would want to see the values where this is set to true only.

Reply:
@dmitri-korin-bcps this still isn't working.   I can't search for Miwa Ito or Anarupa Gupta on the admin/user page.